app/src/js/components/Button/Button.js

```diff
@@ -8,7 +8,8 @@ import {
   Tooltip
 } from 'react-bootstrap';
 
-import './Button.scss';
+// This causes a ParseError
+// import './Button.scss';
```

Comment:
But do we still need this? And if we don't, can we completely get rid of it instead of just commenting it out?

Reply:
Yes, we still need it that's why it's commented out. This is the module issue with CSS and Gulp.